### PR TITLE
[release/2.0.0]  Share common props file for manual shim

### PR DESCRIPTION
**Pull request by weshaggard**
Share common props file for the manual shims.

Add GenFacadesForceZeroVersionSeeds=true so that the assembly references
for the shims are set to 0.0.0.0 so that they will correctly unify
up and support multiple versions of OOB packages like SqlClient.

cc @ericstj 

Fixes https://github.com/dotnet/corefx/issues/28139 in release/2.0.0 branch.

### CODE REVIEW

**Comment by weshaggard** [comment]
Windows CI failures are because of https://github.com/dotnet/corefx/issues/26724.

File: src/shims/manual/dir.props [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap;
+    </BuildConfigurations>
+    <IsNETCoreApp>true</IsNETCoreApp>
+    <IsUAP>true</IsUAP>
+  </PropertyGroup>
+  <!-- need to by-pass the dir.props in the shims directory for this project -->
```
Comment (eerhardt) [inline src/shims/manual/dir.props]: (nit) `this project` doesn't really make sense anymore since this is a `dir.props`.

**Comment by weshaggard** [comment]
FYI @wtgodbe we want to try and get this into the 2.0 servicing release.

**Comment by wtgodbe** [comment]
Do you need me to re-spin a CoreFx 2.0.0 build? I had one succeed earlier today

**Comment by weshaggard** [comment]
>Do you need me to re-spin a CoreFx 2.0.0 build? I had one succeed earlier today 

Yes if the 2.0 corefx build you did was before this change please do respin.

**Pull request merged**